Yoast/wordpress-seo
# Add 'user_language' to links to Yoast.com

## Summary

This PR can be summarized in the following changelog entry:

* Adds 'user_language' as a variable passed in links to yoast.com

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Go to the Yoast SEO dashboard.
* Hover over any link on the page that directs to Yoast.com (i.e. "Improve your SEO skills" > "SEO for Beginners training"). 
* See 'user_language' as a variable in the URL in the browser status bar.

## UI changes

## Documentation

## Quality assurance


Fixes #12140
